docs/contributing/submit-to-plugin-library.md

```diff
@@ -29,6 +29,6 @@ You can include other _relevant_ keywords to your `package.json` file to help in
 ]
 ```
 
-### Images
+### Imágenes
 
-If you include images in your plugin repo's README, please make sure you are referencing the image using an absolute URL in order for the image to show on your plugin page.
+Si incluyes imágenes en tu archivo README del repositorio de tu plugin, por favor asegúrate de referenciar la imagen con una url absoluta de modo que así salga correctamente en la página.
```

Comment:
Mínimo detalle, me parece aquí que debería ser "Si incluyes imágenes el archivo README del repositorio..." 

También que al final sea "que así salga correctamente en la página de dicho plugin" o "en la página de tu plugin". Se que suena redundante, pero sino queda con un significado distinto.

Fuera de eso, ¡muy buena traducción, @xmartinezpujol !

Reply:
El problema que le veo a ese cambio es que quizá no quede demasiado claro el contexto con tales simplificaciones.

En el ejemplo se habla de un caso concreto en el que las imágenes se añaden al README.md, no al proyecto en general. Lo que pasa habitualmente es que en local se ven bien, pero al subir a github/npm los enlaces se rompen por el cambio de path. 

Respecto al último caso es lo mismo, el contexto no es la página del plugin en general, es la página de README.md de la que hemos hablado al inicio.

Por mi parte lo veo claro, ¿qué opináis el resto?

Reply:
Hola muchachos, yo lo veo bien @xmartinezpujol @gusma, le pueden hacer merge cuando quieran :rocket: